bin/git-mr

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -eu
+
+if test "${1-}" = "clean"; then
```

Comment:
`${1-}` could be extracted as a variable. So we don't need to repeat the fallback in each branch.